WordPress/build.gradle

```diff
@@ -469,38 +494,24 @@ configurations.all {
     exclude group: 'org.wordpress', module: 'analytics'
 }
 
-android.buildTypes.all { buildType ->
-    // Add properties named "wp.xxx" to our BuildConfig
-    def inputFile = checkGradlePropertiesFile()
-    def properties = loadPropertiesFromFile(inputFile)
-    properties.any { property ->
-        if (property.key.toLowerCase().startsWith("wp.")) {
-            buildType.buildConfigField "String", property.key.replace("wp.", "").replace(".", "_").toUpperCase(),
-                    "\"${property.value}\""
-        }
-        if (property.key.toLowerCase().startsWith("wp.res.")) {
-            buildType.resValue "string", property.key.replace("wp.res.", "").replace(".", "_").toLowerCase(),
-                    "${property.value}"
-        }
-    }
 
-    // If Google services file doesn't exist...
-    if (!file("google-services.json").exists()) {
-        // ... copy example file.
-        copy {
-            from(".")
-            into(".")
-            include("google-services.json-example")
-            rename('google-services.json-example', 'google-services.json')
-        }
+// If Google services file doesn't exist, copy example file
+if (!file("google-services.json").exists()) {
+    copy {
+        from(".")
+        into(".")
+        include("google-services.json-example")
+        rename('google-services.json-example', 'google-services.json')
     }
+}
 
-    // Print warning message if example Google services file is used.
-    if ((file('google-services.json').text) == (file('google-services.json-example').text)) {
-        println("WARNING: You're using the example google-services.json file. Google login will fail.")
-    }
+// Print warning message if example Google services file is used.
+if ((file('google-services.json').text) == (file('google-services.json-example').text)) {
+    println("WARNING: You're using the example google-services.json file. Google login will fail.")
```

Comment:
I moved this logic about `google-services.json` at the rood of the `build.gradle` file because I don't see the point about doing the copy and the warning multiple times (as it was done inside the loop over each `android.buildTypes.all` before, so once for `debug` and once for `release`), and doing it only once seems enough to me?